src/main/resources/forge-1.20.1/templates/procedure.java.ftl

```diff
@@ -34,48 +34,44 @@ package ${package}.procedures;
 import net.minecraftforge.eventbus.api.Event;
 
 <#assign nullableDependencies = []/>
-<#if !(data.skipDependencyNullCheck)>
-	<#list dependencies as dependency>
-		<#if dependency.getType(generator.getWorkspace()) != "double"
-			&& dependency.getType(generator.getWorkspace()) != "LevelAccessor"
-			&& dependency.getType(generator.getWorkspace()) != "ItemStack"
-			&& dependency.getType(generator.getWorkspace()) != "BlockState"
-			&& dependency.getType(generator.getWorkspace()) != "InteractionResult"
-			&& dependency.getType(generator.getWorkspace()) != "boolean"
-			&& dependency.getType(generator.getWorkspace()) != "CommandContext<CommandSourceStack>">
-			<#assign nullableDependencies += [dependency.getName()]/>
-		</#if>
-	</#list>
-</#if>
-
+<#list dependencies as dependency>
```

Comment:
You completely removed `skipDependencyNullCheck`

Please, test those PRs very well, or I will not have time to test and explain details like this in those plugin PRs. Thanks for understanding.